_templates/exchanges.html

```diff
@@ -181,3 +188,4 @@ <h3 id="united-states"><img src="/img/flags/US.png" alt="United States flag">Uni
 
 <div class="introlink">Visit <a href="https://www.buybitcoinworldwide.com/">Buy Bitcoin Worldwide</a> for
     user reviews on some of the above exchanges.</div>
+<div class="introlink">To find local bitcoin ATMs, you can visit <a href="https://coinatmradar.com/">Coin ATM Radar</a></div>
```

Comment:
Please change this to the following and also add a period at the end of the sentence:

Visit [Coin ATM Radar](https://coinatmradar.com/) to find local Bitcoin ATMs.